lib/ansible/modules/cloud/alicloud/alicloud_instance.py

```diff
@@ -0,0 +1,680 @@
+    disk_category:
+      description:
+        - Category of the system disk.
+      default: "cloud_efficiency"
+      choices: ["cloud_efficiency", "cloud_ssd"]
+      aliases: ["system_disk_category"]
+    disk_size:
+      description:
+        - Size of the system disk, in GB
+      default: 40
+      choices: [40~500]
+      aliases: ["system_disk_size"]
+    disk_name:
+      description:
+        - Name of the system disk.
+      aliases: ["system_disk_name"]
+    disk_description:
+      description:
+        - Description of the system disk.
+      aliases: ["system_disk_description"]
+    count:
+      description:
+        - The number of the new instance. An integer value which indicates how many instances that match the
+          'count_tag' parameter should be running. Instances are either created or terminated based on this value.
+      default: 1
+    count_tag:
+      description:
+      - Used with 'count' to determine how many instances based on a specific tag criteria should be present.
+        This can be expressed in multiple ways and is shown in the EXAMPLES section.
+        The specified count_tag must already exist or be passed in as the 'instance_tags' option.
+        If it is not specified, it will be replaced by 'instance_name' parameter.
+    allocate_public_ip:
+      description:
+        - Whether allocate a public ip for the new instance.
+      default: True
+      aliases: [ 'assign_public_ip' ]
+      type: bool
+    instance_charge_type:
+      description:
+        - The charge type of the instance.
+      choices: ["PrePaid", "PostPaid"]
+      default: "PostPaid"
+    period:
+      description:
+        - The charge duration of the instance. Required when C(instance_charge_type=PrePaid).
+      choices: [1~9,12,24,36]
+      default: 1
+    auto_renew:
+      description:
+        - Whether automate renew the charge of the instance.
+      type: bool
+      default: False
+    auto_renew_period:
+      description:
+        - The duration of the automatic renew the charge of the instance. Required when C(auto_renew=True).
+      choices: [1, 2, 3, 6, 12]
+    instance_ids:
+      description:
+        - A list of instance ids. It is required when need to operate existing instances.
+          If it is specified, 'count' parameter will lose efficacy.
+    force:
+      description:
+        - Whether the current operation needs to be execute forcibly.
+      default: False
+      type: bool
+    instance_tags:
+      description:
+        - A hash/dictionaries of instance tags, to add to the new instance or for starting/stopping instance by tag. C({"key":"value"})
+      aliases: ["tags"]
+    sg_action:
+      description:
+        - The action of operating security group.
+      choices: ['join', 'leave']
+author:
+    - "He Guimin (@xiaozhu36)"
+requirements:
+    - "python >= 2.6"
+    - "footmark >= 1.1.6"
+extends_documentation_fragment:
+    - alicloud
+'''
+
+EXAMPLES = '''
```

Comment:
I'd define the (common) `vars:` just once and list the different examples one after each other within the same `tasks:` section.